libvcx/src/lib.rs

```diff
@@ -341,27 +343,124 @@ mod tests {
     fn test_local_revocation() {
         let _setup = SetupLibraryAgencyV2ZeroFees::init();
 
+        set_institution(None);
         let institution_did = settings::get_config_value(settings::CONFIG_INSTITUTION_DID).unwrap();
-        let (faber, alice) = ::connection::tests::create_connected_connections(None);
+        let (faber, alice) = ::connection::tests::create_connected_connections(None, None);
```

Comment:
Let's try to follow `who_to_whom` connection handle naming convetion from now on